### PR TITLE
feat: Wishlist Management DONE

### PR DESCRIPTION
**Pull request by maitrakhatri**
Please review the page layout and code

Screenshot 2022-02-10 at 19-35-27 gozo Store Log-In

### CODE REVIEW

File: Screens/Wishlist Management/wishlist-management.css [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    margin: 5rem 3rem 0rem 3rem;
+    display: flex;
+    flex-wrap: wrap;
+    gap: 3rem;
+    justify-content: center;
+}
+
+.product-card:hover {
+    filter: drop-shadow(0 0 0.25rem #3333337a);
+}
```
Comment (anupkgurung) [inline Screens/Wishlist Management/wishlist-management.css:23]: try to use the variables where ever possible.

File: Screens/Wishlist Management/wishlist-management.html [hunk 2]
```diff
@@ -0,0 +1,153 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>gozo Store | Log-In</title>
+    <link rel="stylesheet" href="wishlist-management.css">
+</head>
```
Comment (anupkgurung) [inline Screens/Wishlist Management/wishlist-management.html:9]: did you not use your own component library ?

**Review by anupkgurung — COMMENTED** [review]
can't see the component library.

**Review by ianuragsharma — APPROVED** [review]
looks good

File: Screens/Wishlist Management/wishlist-management.html (hunk 2 above)
Comment (maitrakhatri) [reply Screens/Wishlist Management/wishlist-management.html:9]: > did you not use your own component library ?

here's my file structure:

page's CSS file --> import universal CSS(ecom) --> import my component lib

File: Screens/Wishlist Management/wishlist-management.css (hunk 1 above)
Comment (maitrakhatri) [reply Screens/Wishlist Management/wishlist-management.css:23]: > try to use the variables where ever possible.

actually, I don't know we can/how to use variables in shadow. I'll google it. Thanks !!

**Review by anupkgurung — APPROVED** [review]
looks good.

**Pull request merged**